test/OrchardCore.Tests/Localization/PortableObjectStringLocalizerTests.cs

```diff
@@ -200,6 +200,26 @@ public void LocalizerReturnsOriginalTextForPluralIfTranslationDoesntExist(string
             Assert.Equal(expected, translation);
         }
 
+        [Theory]
+        [InlineData("zh-Hans", "zh", "球", 1, new string[] { "球" })]
+        [InlineData("zh-Hans", "zh", "球", 2, new string[] { "球" })]
+        public void LocalizerReturnsCorrectTranslationForPluralIfNoPluralFormsSpecified(string culture, string parentCulture, string expected, int count, string[] translations)
+        {
+            var translationCulture = CultureInfo.GetCultureInfo(parentCulture);
```

Comment:
I expect one culture to be set, please refer to other unit tests

Reply:
Removed